### PR TITLE
Split timetable events out to own class

### PR DESCRIPTION
**Pull request by LucasCairns**

### CODE REVIEW

**Review by davidthomas22 — APPROVED** [review]
LGTM

File: server/services/offender/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const { Offender } = require('./responses/offender');
 const { KeyWorker } = require('./responses/keyWorker');
 const { NextVisit } = require('./responses/nextVisit');
 const { ImportantDates } = require('./responses/importantDates');
-const { TimeTable } = require('./responses/timeTable');
+const { Timetable } = require('./responses/timeTable');
```
Comment (psoleckimoj) [inline server/services/offender/index.js:10]: Can't rename one and not the filename 😛 Timetable is one word so the filename shouldn't be camelcase.

File: server/services/offender/responses/timeTable.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
 const { format, isBefore, addDays, isValid, parseISO } = require('date-fns');
-const { capitalize } = require('../../../utils');
+const { TimetableEvent } = require('./timeTableEvent');
```
Comment (psoleckimoj) [inline server/services/offender/responses/timeTable.js:2]: Ditto here

File: server/services/offender/index.js (hunk 1 above)
Comment (LucasCairns) [reply server/services/offender/index.js]: renamed

File: server/services/offender/responses/timeTable.js (hunk 2 above)
Comment (LucasCairns) [reply server/services/offender/responses/timeTable.js]: renamed

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**